packages/wrapped-keys/src/index.ts

```diff
@@ -1,3 +1,69 @@
-export * from './lib/wrapped-keys';
-export * from './lib/interfaces';
-export { NETWORK_EVM, NETWORK_SOLANA } from './lib/constants';
+import {
+  signMessageWithEncryptedKey,
+  getEncryptedKeyMetadata,
+  exportPrivateKey,
+  generatePrivateKey,
+  importPrivateKey,
+  signTransactionWithEncryptedKey,
+  storeEncryptedKeyMetadata,
+} from './lib/api';
+import { CHAIN_ETHEREUM, LIT_PREFIX } from './lib/constants';
+
+import type {
+  StoredKeyMetadata,
+  SupportedNetworks,
+} from './lib/service-client';
+import type {
+  SignMessageWithEncryptedKeyParams,
+  GetEncryptedKeyMetadataParams,
+  ExportPrivateKeyParams,
+  GeneratePrivateKeyParams,
+  ImportPrivateKeyParams,
+  SignTransactionWithEncryptedKeyParams,
+  ExportPrivateKeyResult,
+  GeneratePrivateKeyResult,
+  EthereumLitTransaction,
+  SerializedTransaction,
+  BaseApiParams,
+  ApiParamsSupportedNetworks,
+  SignTransactionParams,
+  SignTransactionParamsSupportedEvm,
+  SignTransactionParamsSupportedSolana,
+  StoreEncryptedKeyMetadataParams,
+} from './lib/types';
+
+export const constants = {
+  CHAIN_ETHEREUM,
+  LIT_PREFIX,
+};
+
+export const api = {
+  exportPrivateKey,
+  generatePrivateKey,
+  getEncryptedKeyMetadata,
+  importPrivateKey,
+  signMessageWithEncryptedKey,
+  signTransactionWithEncryptedKey,
+  storeEncryptedKeyMetadata,
+};
```

Comment:
Curious why we're wrapping this in an object, does this make it not tree shakeable if a user is only using one or a few of these methods?

Reply:
@MaximusHaximus ^

Reply:
It's still tree-shakeable; bundlers will only include the values from the exported object that the consumer actually uses, and these are all specific functions, rather than classes with mixed dependencies.

I actually had considered adding a `createClient(litNodeClient, sessionSigs})` method to the module exports, which would eliminate those 2 arguments from _all of our API functions_, which would simplify all of our typescript types _and_ make our function signature more concise and easy to understand -- but decided not to because the return value of `createClient()` would _not_ have been tree-shakeable and would have always included all of the functions.

Reply:
If both are tree-shakable then what's the reason behind `export const api = {..}` instead of directly exporting the individual functions? @MaximusHaximus